pom.xml

```diff
@@ -333,12 +333,12 @@
             <version>4.13.1</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
+        <!-- <dependency>
```

Comment:
What is the reason for commenting this out? We need this dependency for the integration tests.

Reply:
With my settings.xml (need some special enterprise setup) I couldn't download the dependency and therefore I had to disable that temporarily and forgot to re-enable it again. Sorry for the confusion.

Did all the things described and reverted the unnecessary changes. I hope it's better now.